docs/raw/9-blockchain-interactions.md

```diff
@@ -0,0 +1,186 @@
+---
+permalink: /spec/9
+parent: Raw specs
+title: 9/BLOCKCHAIN-USAGE
```

Comment:
```suggestion
title: 9/ETHEREUM-USAGE
```

Not 100% sure about this one, either Ethereum-usage or Ethereum-blockchain-usage or Blockchain-usage